src/views/ApiKeyView.py

```diff
@@ -0,0 +1,42 @@
+from flask import request, json, Response, Blueprint, jsonify
+from marshmallow import ValidationError
+from sqlalchemy import exc
+from ..models.ApiKey import ApiKey, ApiKeySchema
+from . import custom_response, api_key_check
+import secrets
+
+
+api_key_api = Blueprint('api_key_api', __name__)
+api_key_schema = ApiKeySchema()
+
+
+@api_key_api.route('/', methods=['POST'])
+def create():
+  try:
+    new_key = secrets.token_urlsafe(10)
+    req_data = request.get_json()
+    req_data['key'] = new_key
+    data = api_key_schema.load(req_data)
```

Comment:
request comes in with just email. generate a new key with `secrets`, put it in the request_data